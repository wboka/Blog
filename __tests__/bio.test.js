import React from "react";
import renderer from "react-test-renderer";

import Bio from "../src/components/bio";

describe("Bio", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<Bio className="hello" />).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
