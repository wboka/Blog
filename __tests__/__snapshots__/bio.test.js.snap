// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders correctly 1`] = `
<div
  className="hello"
>
  <img
    alt="Headshot of Wayne Boka, Web Developer"
    className="rounded-full w-1/2 sm:w-1/4 my-2 mx-auto shadow-lg"
    onClick={[Function]}
    onTouch={[Function]}
    src="test-file-stub"
  />
  <p>
    Hello! My name is Wayne and I am a website developer based out of Westfield, PA. I have been building websites and web applications since 2011.
     
    <a
      className="border-b-4 border-blue-500 hover:bg-blue-500 px-2 pt-2 no-hover rounded inline-block"
      href="https://m.me/bokasolutions"
    >
      Send me a Facebook message
    </a>
     
    or
     
    <a
      className="border-b-4 border-blue-500 hover:bg-blue-500 px-2 pt-2 no-hover rounded inline-block"
      href="mailto:wayne@bokasolutions.com"
    >
      email me
    </a>
     
    to discuss how I can help build your online presence.
  </p>
</div>
`;
